tests/sls/homu/valid_repos.py

```diff
@@ -0,0 +1,39 @@
+from urllib.error import URLError
+from urllib.request import Request, urlopen
+
+import toml
+
+from tests.util import Failure, Success
+
+
+def repoExists(identifier):
```

Comment:
It's more Pythonic to use snake_case: `repo_exists`.

Also, I think `repo` is more clear than `identifier`.